stories/setup/Provider.js

```diff
@@ -0,0 +1,16 @@
+/* @flow */
+import { createBrowserHistory } from 'history';
+import React from 'react';
+import { Provider as ReduxProvider } from 'react-redux';
+
+// Without core/polyfill we see the following error:
+// "regeneratorRuntime is not defined".
+import 'core/polyfill';
```

Comment:
Would it be possible to put this in a more central entry point? I worry that `setup/Provider` might be too late in the bundling process. What if a module that comes before it needs the polyfill? Putting it in `setup/webpack.config.babel.js` seemed like a central entry point. Does that no longer work?

Reply:
It didn't work :/ but let me trying moving it maybe to config or somewhere higher in the chain if I can..